com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -1322,31 +1324,41 @@ private bool ExportInstance(GameObject unityGo, FbxScene fbxScene, FbxNode fbxNo
             {
                 return false;
             }
-            
-            PrefabAssetType unityPrefabType = PrefabUtility.GetPrefabAssetType(unityGo);
-            // only export as an instance if the GameObject is part of a prefab instance
-            if (unityPrefabType == PrefabAssetType.MissingAsset ||
-                unityPrefabType == PrefabAssetType.NotAPrefab ||
-                PrefabUtility.GetPrefabInstanceStatus(unityGo) != PrefabInstanceStatus.Connected)
-            {
-                return false;
-            }
```

Comment:
Why don't you need these checks anymore?

Reply:
This check is checking if the object is a prefab, but now non-prefabs can also be exported as instances so it's not needed anymore. Later on, it'll return false if the object is neither a prefab nor a mesh that's been exported before.